packages/odyssey/src/scss/components/_checkbox.scss

```diff
@@ -14,15 +14,26 @@
 
 /* stylelint-disable no-descending-specificity, max-nesting-depth */
 
+$size-checkbox: 1em / 0.875;
```

Comment:
Why are we computing this in some places while others we rely on actual decimal values?